src/renderers/shared/fiber/ReactFiberStack.js

```diff
@@ -34,6 +34,21 @@ exports.createCursor = function<T>(defaultValue : T) : StackCursor<T> {
   };
 };
 
+exports.getPrevious = function<T>(
+  cursor : StackCursor<T>,
+) : T | null {
+  if (index < 0) {
+    if (__DEV__) {
+      warning(false, 'Unexpected read of previous.');
+    }
+    return null;
+  }
+  if (index === 0) {
```

Comment:
This check is unnecessary. We only care about < 0 which is already checked for above.